shared/util.go

```diff
@@ -334,11 +340,33 @@ func ParseLXDFileHeaders(headers http.Header) (*LXDFileHeaders, error) {
 		return nil, fmt.Errorf("Invalid file write mode: %q", write)
 	}
 
+	UIDModifyExisting := false
```

Comment:
just for future reference (no change required) you can also do:

```go
var UIDModifyExisting, GIDModifyExisting, modeModifyExisting bool
```

Also as we're not exporting the vars they should be `uidModifyExisting` (i.e we only capitalise all of them if we are exporting the var and so must capitalise the first letter).